simple_parsing/wrappers/field_wrapper.py

```diff
@@ -1040,3 +1028,87 @@ def only_keep_action_args(options: Dict[str, Any], action: Union[str, Any]) -> D
         logger.debug(f"Kept options: \t{kept_options.keys()}")
         logger.debug(f"Removed options: \t{deleted_options.keys()}")
     return kept_options
+
+
+_BOOL_NO_PREFIX = "--no"
+
+
+class _BooleanOptionalAction(argparse.Action):
+    """Similar to `argparse.BooleanOptionalAction`.
+
+    * Support custom prefix (for `absl.flags` compatibility)
+    * Accept `--flag=true` value
+    * Support `nargs`
+    * Support Python 3.8
+
+    """
+
+    def __init__(
+        self,
+        option_strings,
+        dest,
+        default=None,
+        type=None,
+        choices=None,
+        required=False,
+        help=None,
+        metavar=None,
+        nargs=None,
```

Comment:
Sure, then I suggest using `int | Literal["?", "*", "+"] | None`

Reply:
Pushed my changes. Not that `str` is better than `Literal` here because there's some undocumented argparse value for `nargs`. Currently, those will raise error.

Reply:
What value? In which case would it raise an error?

Reply:
I'm talking of `argparse.REMAINDER`, `argparse.PARSER`...: https://github.com/python/cpython/blob/cb6ff54b01118e01c26758d2f57cf6da60277bd4/Lib/argparse.py#L101

No idea what those are doing. Currently this will raise an explicit error message in:

`if not (isinstance(nargs, int) or nargs in {'?', '*', '+'}):` would raise an error.

I think this is fine for now. If really required, we could add support for those later.